02LessonTwo/homework/css/style.css

```diff
@@ -0,0 +1,35 @@
+#name {
+      width: 100%;
+      background: orange;
+      text-align: center;
+      color: black;
+      font-size: 50px;
+      font-family: Helvetica;
+      border-radius: 10px;
+      margin-left:20px;
+      margin-right: 20px;
+}
+ .facts {
+     width: 35%;
+     background-color: #ebf090;
+     color; black;
+     border-radius: 5px;
+     text-align: center;
+     margin: 5px;
+     padding: 25px;
+   }
+
+  .random {
+      width: 35%;
+      background-color: #ebf090;
+      color; black;
+      border-radius: 5px;
```

Comment:
Excellent effort on writing the CSS. You will get much better with practice. You have the drive and understand how to target HTML elements. :+1: